bin/stage

```diff
@@ -33,15 +32,36 @@
 # Configuration
 # ========================================
 
+# Load in configuration file, or ask and populate.
+def read_or_ask_for_configuration
+  require 'yaml'
+  conf_file = File.join(ENV['HOME'], '.jekyll-stagingrc')
```

Comment:
Hi Parker,

Thanks so much. I always run the script from the site's Jekyll root anyway (since I'll then be invoking the preview server), so I think it makes complete sense to use the working directory.